### ISSUE TITLE
Reading .webm

### ISSUE THREAD

**Issue by stefan-reich** [jcodec/jcodec#166]
Since Stan has been so helpful with the last issues, here's another one... I can't grab frames from a .webm file:

`java.lang.NullPointerException
	at org.jcodec.api.FrameGrab8Bit.createFrameGrab8Bit(FrameGrab8Bit.java:49)`

Greetings :-)

**Comment by svitvitskiy** [comment]
Yes, FrameGrab doesn't support WebM yet. Transcoder does support a very limited WebM/VP8 I-frame only case.
A couple of things need to happen:
  - FrameGrab needs to be refactored to support everything Transcode supports;
  - VP8 decoder has to be extended to support P-frames as well;
  - VP9 decoder has to be implemented.

Please leave this bug open, I will post the progress. Just heads up, don't expect this soon :)

**Comment by stefan-reich** [comment]
Ah. I checked vokoscreen and it can also generate .avi. But FrameGrab doesn't do that either?

I guess I'll always have to convert with ffmpeg first... it's a bit unfortunate.

**Comment by svitvitskiy** [comment]
AVI demuxer has been contributed but it's not integrated yet.

**Comment by stefan-reich** [comment]
Will it be integrated?

.mkv to .mp4 seems to be working, which SHOULD solve my problems. I want an automated chain from all my sources (vokoscreen and YouTube videos) through the AI-based video processing I am making.

**Comment by svitvitskiy** [comment]
What's inside the AVI? h264?

**Comment by stefan-reich** [comment]
Uh... yeah. In vokoscreen it says "Videocodec: libx264".

**Comment by svitvitskiy** [comment]
Ok, so one more request for this thread: integrate AVI demuxer. I'll post here, again, as with everything else, don't expect it soon.

**Comment by stefan-reich** [comment]
OK then I have to find quicker solutions :-) Are you doing this full-time?! Haha... I try to do my stuff full time :-)
